ROADMAP.md

```diff
@@ -1,10 +1,54 @@
 # Kubeflow Roadmap
 
-## Kubeflow 1.5 Release, Due: February 2022
-* Kubeflow 1.5 timeline is tracked in [#538](https://github.com/kubeflow/community/pull/538)
-* Additional details will be posted in early 1Q'22
+## Kubeflow 1.5 Release, Due: March 2022
+* Kubeflow 1.5 [milestones and timeline](https://github.com/kubeflow/community/pull/538)
+
+### Themes
+* Switching to [Emissary executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor) enables Kubeflow Pipelines deployment on Kubernetes >= v1.20, which runs on containerd runtime instead of Docker runtime
+* Improve model accuracy and reduce overfitting, especially with hyper parameter tuning
+* Simplify operations and optimize utilization (including spot instance use cases for distributed training) 
+* More consistent user experience - UI appearance, features and naming
+* Improved documentation, tutorials and examples
+* Stretch - Support for K8s 1.22 and associated dependencies (cert mgr, istio)
+
+### Major Features from each Working Group (note: Individual WG versions are independent of Kubeflow's)
+
+#### Kubeflow Pipelines, v1.8
+* Switching to [Emissary executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor) enables Kubeflow Pipelines deployment on Kubernetes >= v1.20, which runs on containerd runtime instead of Docker runtime.
+
+#### Katib, v0.13 
+* Hyperparameter leader election for HA operations and faster recovery
+* Validation for Early Stopping algorithm settings improves accuracy, reduces overfitting
```

Comment:
Should we modify this @jbottum @tenzen-y @johnugeorge ?
```suggestion
* Validation for Early Stopping algorithm settings helps users to proper reduce model overfitting
```

Reply:
It sounds great to me @andreyvelich!

Reply:
@andreyvelich do we need the word "proper"

Reply:
@jbottum Basically, validation helps to avoid user's errors and mistakes in APIs when they use Early Stopping.
That is why I was thinking about this change.

Reply:
@jbottum understand but I am not sure this correct - helps users to **proper** reduce model overfitting....shouldn't it read...helps users to reduce model overfitting ?

Reply:
@andreyvelich I put this in another commit lower down, need to move this forward.